Formula/pyqt-builder.rb

```diff
@@ -0,0 +1,18 @@
+class PyqtBuilder < Formula
```

Comment:
Is this formula useful for anything other than building `pyqt`?

If it isn't, it might be better to just include this in `pyqt` as a `resource`.

Reply:
`qscintilla2` need it